packages/kit/src/runtime/client/client.js

```diff
@@ -1016,7 +1007,10 @@ export function create_client(app, target) {
 		url = intent?.url || url;
 
 		// abort if user navigated during update
-		if (token !== nav_token) return false;
+		if (token !== nav_token) {
+			nav.reject(new Error('navigation was aborted'));
```

Comment:
Feels a bit loose to just return strings here. Should we declare a set of `const`s that are reused? And should it just be "navigation was aborted" or a more techincal/stable-looking thing like `NAV_ABORTED`?

```js
const NAV_ABORTED = 'NAV_ABORTED';
const NAV_CANCELLED = 'NAV_CANCELLED';
```

Reply:
Are you saying the error message part of the public API and the user needs to check which error message is present?